### PR TITLE
Feature/pelagos 5057 re order dataland

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: templates/Dataland/v2/index.html.twig [hunk 1]
```diff
@@ -38,7 +38,7 @@
                             <div>
                                 <div>
                                     <div>
-                                        <strong>DOI:</strong> {{ dataset.doi.doi|default }}
+                                        <strong>DOI:</strong> {{ dataset.doi.doi|default|formatWebDoi }}
```
Comment (mickel1138) [inline templates/Dataland/v2/index.html.twig:41]: Why not just format the URL here? Rather than using a whole function?

Comment (fightingtexasaggie) [reply templates/Dataland/v2/index.html.twig:41]: yeah, you're right. We probably don't need to load more into twig.

File: templates/Dataland/v2/index.html.twig [hunk 2]
```diff
@@ -38,7 +38,10 @@
                             <div>
                                 <div>
                                     <div>
-                                        <strong>DOI:</strong> {{ dataset.doi.doi|default }}
+                                        <strong>DOI:</strong>
+                                        {% if dataset.doi %}
+                                            https://doi.org/{{ dataset.doi.doi|default }}
```
Comment (mickel1138) [inline templates/Dataland/v2/index.html.twig:43]: default would be null, thats not a good url...

**Review by mickel1138 — APPROVED** [review]
LGTM

**Pull request merged**